pkg/controller/machinepool/awsactuator.go

```diff
@@ -87,6 +89,105 @@ func NewAWSActuator(
 	return actuator, nil
 }
 
+// GenerateCAPIMachineSets takes a clusterDeployment and returns CAPI MachineSets
+func (a *AWSActuator) GenerateCAPIMachineSets(cd *hivev1.ClusterDeployment, pool *hivev1.MachinePool, logger log.FieldLogger) ([]*capiv1.MachineSet, bool, error) {
+	computePool := baseMachinePool(pool)
```

Comment:
Do we need an installer compute pool here?

Reply:
I'm not sure we do.

Reply:
This comment is so old I'm not even sure what I was thinking. :)